packages/yasgui/src/Tab.ts

```diff
@@ -195,19 +194,18 @@ export class Tab extends EventEmitter {
 
   private checkEndpointForCors(endpoint: string) {
     if (this.yasgui.config.corsProxy && !(endpoint in Yasgui.corsEnabled)) {
-      superagent
-        .get(endpoint)
-        .query({ query: "ASK {?x ?y ?z}" })
-        .then(
-          () => {
+      fetch(`${endpoint}?${new URLSearchParams({ query: "ASK {?x ?y ?z}" }).toString()}`)
```

Comment:
The final URL might not work if the endpoint already contains some query parameters.